core/src/scene/dataLayer.cpp

```diff
@@ -2,9 +2,9 @@
 
 namespace Tangram {
 
-DataLayer::DataLayer(SceneLayer _layer, const std::string& _source, const std::string& _collection) :
+DataLayer::DataLayer(SceneLayer _layer, const std::string& _source, const std::vector<std::string>& _collections) :
     SceneLayer(_layer),
     m_source(_source),
-    m_collection(_collection) {}
+    m_collections(_collections) {}
```

Comment:
We can do a `std::move` here.

P.S. github app does not respect where you comment :(.